app/views/users/show.html.erb

```diff
@@ -1,9 +1,23 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <ul>
+    <li>
+      <% if current_user.friend_invites(@user) %>
+        <button class="btn btn-warning"> Invitation sent </button>
+      <% elsif current_user.receive_invitation(@user) %>
+        <%= link_to 'Accept', accept_path(user_id: @user.id), class:"btn btn-primary" %>
+        <%= link_to 'Decline', reject_path(user_id: @user.id), method: :delete, class:"btn btn-danger" %>
+      <% elsif !current_user.friend?(@user) && current_user.id != @user.id %>
+        <%= link_to 'Add Friend', invite_path(user_id: @user.id) , class:"btn btn-success" %>
+      <% elsif current_user.friend?(@user)  %>
+        <button class="btn btn-warning"> A friend </button>
+      <% end %>
```

Comment:
remove logic from views and move them into view helpers

Reply:
Logic has been moved to helpers.

Thanks